emsdk.py

```diff
@@ -355,21 +358,9 @@ def win_get_environment_variable(key, system=True, fallback=True):
   return value
 
 
-def win_environment_variable_exists(key, system=True):
-  value = win_get_environment_variable(key, system)
-  return value is not None and len(value) > 0
-
-
-def win_get_active_environment_variable(key):
-  value = win_get_environment_variable(key, False)
-  if value is not None:
-    return value
-  return win_get_environment_variable(key, True)
-
-
-def win_set_environment_variable(key, value, system=True):
+def win_set_environment_variable(key, value, system=True, user=True):
```

Comment:
Maybe remove the default parameters here since we actually want to require all these argument everywhere now right?

Reply:
Fixed in https://github.com/emscripten-core/emsdk/pull/642/commits/e7d8a37db14f089451adaa23f3006db704d861d5